package.json

```diff
@@ -47,6 +46,7 @@
     "truffle-hdwallet-provider": "0.0.3"
   },
   "dependencies": {
+    "homedir": "^0.6.0",
```

Comment:
wat

Reply:
🤔

Reply:
idk

Reply:
wish i remembered

Reply:
wish i remembered

Reply:
l think related to being able to use it from aragon-apps, whose truffle.js just requires aOS truffle.js

Reply:
Maybe we should move the truffle config out into a package if we're going to reuse it like this.